network-api/networkapi/wagtailpages/rss.py

```diff
@@ -18,7 +18,7 @@ class RSSFeed(Feed):
     description = 'The Mozilla Foundation Blog'
 
     def items(self):
-        blog_index = IndexPage.objects.get(title__iexact='blog')
+        blog_index = BlogIndexPage.objects.get(title__iexact='Blog')
```

Comment:
does this need to be locale-aware (e.g., `title_en__iexact`)?

Reply:
hm, yeah probably. We have a "get _en, if that doesn't exist, try plain" somewhere else, let me put that same logic here.